<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [FunctionalityTesting]" guice-stage="DEVELOPMENT" verbose="0">
  <listeners>
    <listener class-name="Utilities.TestReportListener"/>
  </listeners>
  <test thread-count="5" name="FunctionalityTesting(failed)" verbose="0">
    <classes>
      <class name="PaymentTestCases.TC16_CloseTheguaranteeWhichIsAlreadyInvoked">
        <methods>
          <include name="setup"/>
          <include name="runTest"/>
        </methods>
      </class> <!-- PaymentTestCases.TC16_CloseTheguaranteeWhichIsAlreadyInvoked -->
      <class name="PaymentTestCases.TC15_Invokeguaranteeaftermarkingitforinvocation">
        <methods>
          <include name="runTest"/>
          <include name="setup"/>
        </methods>
      </class> <!-- PaymentTestCases.TC15_Invokeguaranteeaftermarkingitforinvocation -->
    </classes>
  </test> <!-- FunctionalityTesting(failed) -->
</suite> <!-- Failed suite [FunctionalityTesting] -->
